core/src/main/java/hudson/model/TopLevelItemDescriptor.java

```diff
@@ -114,6 +127,64 @@ public String getDisplayName() {
     }
 
     /**
+     * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
+     * in order to avoid how it should be represented.
+     *
+     * @return A string, an empty string by default.
```

Comment:
@jglick What are you seeing? :dizzy_face: I updated the javadoc for that:

```
/**
 * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
 * in order to avoid how it should be represented.
 *
 * This method should be called in a thread where Stapler is associated, but it will return an empty string.
 *
 * @return A string, by default the value from newInstanceDetail view is taken.
 *
 * @since TODO
 */
```